app/code/Inchoo/Sample08/etc/di.xml

```diff
@@ -4,4 +4,13 @@
     <type name="Inchoo\Sample08\Model\Calculator">
         <plugin name="inchoo_sample08" type="Inchoo\Sample08\Plugin\CalculatorPlugin"/>
     </type>
+    <type name="Magento\Catalog\Model\ResourceModel\Product\Collection">
+        <plugin name="inchoo_sample08" type="Inchoo\Sample08\Plugin\CollectionPlugin"/>
+    </type>
```

Comment:
nedostaje ti:
"Use plugin `sortOrder` to run the custom plugin before Magento's `add_stock_information` plugin."